test/integration/test_groundlight.py

```diff
@@ -6,13 +6,15 @@
 
 import openapi_client
 import pytest
+
 from groundlight import Groundlight
 from groundlight.binary_labels import VALID_DISPLAY_LABELS, DeprecatedLabel, Label, convert_internal_label_to_display
 from groundlight.internalapi import NotFoundError
 from groundlight.optional_imports import *
 from groundlight.status_codes import is_user_error
 from model import ClassificationResult, Detector, ImageQuery, PaginatedDetectorList, PaginatedImageQueryList
 
+
```

Comment:
I would just delete these added lines, that should fix the linter. Not sure why the auto-format bot isn't working.